code/src/components/RadioForm.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+const RadioForm = ({input, setInput}) => {
+    const rolesList = [
+        'Frontend',
+        'Backend',
+        'Fullstack',
+        'Cloud'
+    ]
+
+    return (
+        <div className="radio-form-div">
```

Comment:
It doesn't seem like you use `className="radio-form-div"` in your stylesheet. Do you think it's better if you remove it, if you are not using it?